src/lib/mongoBackend/MongoCommonSubscription.cpp

```diff
@@ -307,7 +307,8 @@ void setCondsAndInitialNotify
   const std::string&               fiwareCorrelator,
   BSONObjBuilder*                  b,
   bool*                            notificationDone,
-  ApiVersion                       apiVersion
+  ApiVersion                       apiVersion,
```

Comment:
I'd suggest to interchange the position of apiVersion and skipInitialNotification parameters. The apiVersion parameters is typically used as last one in the functions requiring so.

Reply:
fixed in a3d00ef